### DISCUSSION TITLE
Can we delete v1 db after migration is completed v2. while upgrading 1.3.0

### DISCUSSION THREAD

**Discussion by obulisaravanan** [Chia-Network/chia-blockchain#10582] [Support]
Can we delete v1 db after migration is completed v2. while upgrading 1.3.0

**Comment by loppefaaret** [comment]
After successfully upgrading your v1 DB to v2 from the CLI, you can remove the v1 database, yes

**Reply by obulisaravanan** [reply]
How to do migration on Windows GUI

**Reply by henriklg** [reply]
> How to do migration on Windows GUI

@obulisaravanan check out [this article](https://thechiaplot.net/2022/02/11/chia-database-upgrade-process-and-results/) from the 1.3beta (the process is still pretty much the same) or the [wiki pages](https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-new-database)
